cluster/gce/gci/configure.sh

```diff
@@ -586,19 +586,23 @@ function ensure-container-runtime {
   if [[ -n "${UBUNTU_INSTALL_CONTAINERD_VERSION:-}" || -n "${UBUNTU_INSTALL_RUNC_VERSION:-}" ]]; then
     log-wrap "InstallContainerdUbuntu" install-containerd-ubuntu
   fi
+
+  # source containerd env if exists
```

Comment:
can you please update comment suggesting what this would mean. Like this:

```suggestion
  # when custom containerd version is installed sourcing containerd_env.sh will add all tools like ctr to the PATH
```

Reply:
Done.